crates/rome_js_formatter/src/formatter.rs

```diff
@@ -203,6 +203,36 @@ impl Formatter {
         )
     }
 
+    /// Formats a group delimited by an opening and closing token, placing the
+    /// content in an [indent] group with [hard_line_break] tokens at the
+    /// start and end
+    pub(crate) fn format_delimited_block_hard_line_break(
+        &self,
+        open_token: &JsSyntaxToken,
+        content: FormatElement,
+        close_token: &JsSyntaxToken,
+    ) -> FormatResult<FormatElement> {
+        self.format_delimited(
+            open_token,
+            move |trailing_trivia, leading_trivia| {
+                if trailing_trivia.is_empty() && content.is_empty() && leading_trivia.is_empty() {
+                    empty_element()
+                } else {
+                    format_elements![
+                        indent(format_elements![
+                            hard_line_break(),
+                            trailing_trivia,
+                            content,
+                            leading_trivia,
+                        ]),
+                        hard_line_break(),
```

Comment:
You can use `block_indent` here: 
```suggestion
                        block_indent(format_elements![
                            trailing_trivia,
                            content,
                            leading_trivia,
                        ]),
```

Reply:
Apart from the initial check, this function is identical to `format_delimited_block_indent`: https://github.com/rome/tools/blob/main/crates/rome_js_formatter/src/formatter.rs#L144-L157

Also, the `is_empty` check is already done inside `Formatter::format_delimited`, so it's not needed. And check `content.is_empty` is a bit... weird, why would a developer pass an empty `FormatElement`?

Reply:
Updated to use `format_delimited_block_indent`, I wasn't aware of how it worked! For context, I previously mostly just copy-pasted `format_delimited_soft_block_indent`.